src/main/java/io/seqera/tower/cli/commands/global/WorkspaceOptions.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.commands.global;
+
+import picocli.CommandLine;
+
+public class WorkspaceOptions {
+
+    @CommandLine.Option(names = {"-w", "--workspace"}, description = "Workspace numeric identifier (TOWER_WORKSPACE_ID)", defaultValue = "${TOWER_WORKSPACE_ID}")
```

Comment:
When this `mixin` is added the workspace is always required. I think we need to set `required = true` here.

Reply:
If I make it required, it will still read the env if omitted but will throw an error if the env is not set, which is not desirable when working with user's resources.

Reply:
What do you mean "working with user's resources"?

Reply:
A compute environment that is not under a given workspace, for instance.

Reply:
Ok, I see. Yes, you're right. Better to keep it optional.